test/conformance/tests/test_exporter_metrics.go

```diff
@@ -19,21 +19,18 @@ import (
 	"github.com/prometheus/common/model"
 
 	corev1 "k8s.io/api/core/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 )
 
-var _ = Describe("[sriov] Metrics Exporter", Ordered, func() {
+var _ = Describe("[sriov] Metrics Exporter", Ordered, ContinueOnFailure, func() {
```

Comment:
using Skip() is not enough in Ordered containers ?

Reply:
The problem with Ordered containers is that if a test fails, all the subsequent tests are skipped:

https://pkg.go.dev/github.com/onsi/ginkgo/v2#pkg-constants
> ContinueOnFailure is a decorator that allows you to mark an Ordered container to continue running specs even if failures occur. Ordinarily an ordered container will stop running specs after the first failure occurs.

Reply:
ack, thx for the info.